src/stcal/jump/jump.py

```diff
@@ -427,6 +427,13 @@ def detect_jumps(
             # save the neighbors to be flagged that will be in the next slice
             previous_row_above_gdq = row_above_gdq.copy()
             k += 1
+            # remove redundant bits in pixels that have jump flagged but were
+            # already flagged as do_not_use or saturated.
+            gdq[gdq == np.bitwise_or(dqflags['DO_NOT_USE'], dqflags['JUMP_DET'])] = \
```

Comment:
Do you really want to use `bitwise_or` here, as opposed to `bitwise_and`? Using `or` will result in pixels that have the JUMP_DET flag set, but not DO_NOT_USE, getting reset to DO_NOT_USE. I think you only want to operate on pixels that have *both* flags set. Same down in line 434, for saturation.

Reply:
The bitwise_or combination is done first. This combines JUMP_DET with either SATURATION or DO_NOT_USE. So, this is looking for pixels in the gdq that have values of exactly 5 or 6. 
I did test with and without these changes and with the changed code the gdq cubes are identical. Without these two lines there were 5 and 6 gdq values that in the single processing results were 1 and 4 respectively.